### PR TITLE
Linting some more whitespace and defining rules

### PR DESCRIPTION
**Pull request by fabianmoronzirfas**
I took on the next file for linting.

### CODE REVIEW

File: src/includes/constants.js [hunk 1]
```diff
@@ -295,4 +295,4 @@ pub.DEFAULTMODE = pub.MODEVISIBLE; // FIXME, DEFAULTMODE shouldn't be public, mo
 
 
 var ERROR_PREFIX = "\nBasil.js Error -> ",
-    WARNING_PREFIX = "### Basil Warning -> ";
\ No newline at end of file
+  WARNING_PREFIX = "### Basil Warning -> ";
```
Comment (trych) [inline src/includes/constants.js]: Hm, is there any chance we can leave this sort of formatting? I think it looks just cleaner, if it is aligned. In this specific case it's not such a drama, but what about cases like [this one](https://github.com/basiljs/basil.js/blob/master/src/includes/environment.js#L657). Would it not be possible to keep those?

Just my personal opinion, if others disagree, go ahead and lint it like this.

**Comment by fabianmoronzirfas** [comment]
Hm. Something is creating a conflict. 😢  I'll investigate

**Comment by fabianmoronzirfas** [comment]
I took a look into the diff of those files. it is all just whitespace, quotes and some unneeded ;.  

All test are passing on a fresh bundled version


    Indesign returned: 53 test cases run
    239 assertions run
    239 passed
    0 failed

If it is okay with @b-g and @ffd8 I will merge this by myself

**Comment by trych** [comment]
Just don't forget to bundle the file before you merge. :wink:

**Comment by fabianmoronzirfas** [comment]
Okay. I fixed all those conflicts.

**Comment by b-g** [comment]
Many thanks for this! Wow, so tidy and clean!

**Comment by fabianmoronzirfas** [comment]
So I have the go to merge?

**Comment by b-g** [comment]
Sorry for not being clear! Yes please go ahead.

**Pull request merged**